.bazelrc

```diff
@@ -46,6 +46,8 @@ build:ci --build_metadata=ROLE=CI --workspace_status_command=./scripts/workspace
 build:ci --local_cpu_resources=8
 build:ci --local_ram_resources=15000
 build:ci --config="release"
+# Circle macos.m1.large.gen1 8 cores 12gb ram
+common:ci-mac --config=ci --local_cpu_resources=8 --local_ram_resources=11000
```

Comment:
do we have any guidelines/best practices on what we can use so we dont go over budget? Or is this just a by case basis. Knowing me I would just try to use a machine with maxed out ram and cores so builds are fast; but cant imagine how much that would cost

Reply:
Builds use up "credits", and the macOS executors use more credits per minute, so its more case by case, and more important for iOS related things, docker credits are cheap

You can see some credit usage here: https://app.circleci.com/insights/github/player-ui

Reply:
the bazel remote cache should also aid in build times without needing to increase CI machine resources